<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pixivDownloader</class>
 <widget class="QWidget" name="pixivDownloader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pixivDownloader</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
	color:#ff0000
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="edit_userid">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_download">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Download</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_status">
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
